lizard_auth_server/urls.py

```diff
@@ -92,6 +93,17 @@ def check_settings():
         views_sso.VerifyView.as_view(),
         name='lizard_auth_server.sso.api.verify'),
 
+    # The (newer) V2 API
```

Comment:
Als je een naam van een URL niet goed vindt is dit dus het moment om er eentje te veranderen. Van "logout_redirect" is me niet direct duidelijk wat 'ie ook al weer doet, bijvoorbeeld.